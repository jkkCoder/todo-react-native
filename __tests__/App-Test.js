import "react-native"
import React from "react"
import Header from "../components/Header"
import renderer from "react-test-renderer"

test("Header snapshot",()=>{
    const snap = renderer.create(<Header />).toJSON();
    expect(snap).toMatchSnapshot();
})