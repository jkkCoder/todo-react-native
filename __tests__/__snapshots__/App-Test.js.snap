// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "coral",
      "height": 80,
      "paddingTop": 38,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
        "fontSize": 20,
        "fontWeight": "bold",
        "textAlign": "center",
      }
    }
  >
    My Todos
  </Text>
</View>
`;
